src/app/app.component.ts

```diff
@@ -9,15 +8,9 @@ import { ErrorsService } from './errors.service';
 export class AppComponent {
   teste: string = ''
 
-  constructor(private errorsService:ErrorsService) {  }
+  constructor() {  }
   title = 'angularProject';
```

Comment:
adiciona a visibilidade (public, private, ...) na classe (querendo ou nao o componente é uma classe) em variaveis, e não esquece de dizer qual o tipo de dado... No lugar dessa linha dava pra trocar por:
```suggestion
public title: string = 'angularProject';
```